orion/sites/admin/src/home/home.js

```diff
@@ -2,30 +2,211 @@
+            this.setState({
+                noUnpub: "All items are published! Your website is up to date!",
+                switch: false,
+            });
+        }
+    }
+
+    onClickClasses() {
+        let classes = this.state.classes.map((row, index) => {
+            return <DashboardRow key={index} title={row.classId} />;
+        });
+        this.setState({
+            noUnpub: "",
+            rows: classes,
+            selection: "classes",
+            switch: true,
+        });
+        if (classes.length == 0) {
+            this.setState({
+                noUnpub: "All items are published! Your website is up to date!",
+                switch: false,
+            });
+        }
+    }
+
+    onClickLocations() {
+        let locations = this.state.locations.map((row, index) => {
+            return <DashboardRow key={index} title={row.locId} />;
+        });
+        this.setState({
+            noUnpub: "",
+            rows: locations,
+            selection: "locations",
+            switch: true,
+        });
+        if (locations.length == 0) {
+            this.setState({
+                noUnpub: "All items are published! Your website is up to date!",
+                switch: false,
+            });
+        }
+    }
+
+    onClickAchievements() {
+        let achieves = this.state.achieves.map((row, index) => {
+            return <DashboardRow key={index} title={row.message} />;
+        });
+        this.setState({
+            noUnpub: "",
+            rows: achieves,
+            selection: "achievements",
+            switch: true,
+        });
+        if (achieves.length == 0) {
+            this.setState({
+                noUnpub: "All items are published! Your website is up to date!",
+                switch: false,
+            });
+        }
+    }
+
+    onClickSemesters() {
+        let semesters = this.state.semesters.map((row, index) => {
+            return <DashboardRow key={index} title={row.semesterId} />;
+        });
+        this.setState({
+            noUnpub: "",
+            rows: semesters,
+            selection: "semesters",
+            switch: true,
+        });
+        if (semesters.length == 0) {
+            this.setState({
+                noUnpub: "All items are published! Your website is up to date!",
+                switch: false,
+            });
+        }
+    }
+
+    onClickSessions() {
+        let sessions = this.state.sessions.map((row, index) => {
+            return <DashboardRow key={index} title={row.sessionId} />;
+        });
+        this.setState({
+            noUnpub: "",
+            rows: sessions,
+            selection: "sessions",
+            switch: true,
+        });
+        if (sessions.length == 0) {
+            this.setState({
+                noUnpub: "All items are published! Your website is up to date!",
+                switch: false,
+            });
+        }
+    }
+
+    onClickPage() {
+        console.log("Go to Page clicked");
```

Comment:
do you think you can actually link to the next page. We've done it before in the other pages.
You use something like window.location.___ = "programs";

Note you do know where to link to based on the this.state.selection variable.